health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/DraggableField.js

```
@@ -0,0 +1,71 @@
+    },
+  });
+
+  const [{ isDragging }, drag] = useDrag({
+    type: FIELD_TYPE,
+    item: { index },
+    collect: (monitor) => ({ isDragging: monitor.isDragging() }),
+  });
+
+  drag(drop(ref));
+  return (
+    <div ref={ref} style={{ opacity: true ? 0.5 : 1, display: "flex", alignItems: "center" }}>
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)